recipe/meta.yaml

```diff
@@ -13,13 +13,11 @@ source:
   sha1: afde507fbee85a1aeb5fcb32e897bd3ea2bc48f5  # [win]
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - python
-    - python.app  # [osx]
```

Comment:
Not sure if this is OK. but there is no `python.app` for `PY36` in `defaults` and we do not have that in `conda-forge`. (Not even sure if we need it. @ccordoba12 do you own a Mac? Can you chime in here and let me know if I should do it differently?)